backend/base/templates/loading_page.html

```diff
@@ -26,7 +26,7 @@
             grid-auto-rows: 130px;
             place-items:center;
         }
-        
+

```

Comment:
any reason for this change?

Reply:
I removed the unwanted spaces to improve the code.